mautic/mautic
# My activity on user profile + better accountability for user view

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This enhancement aims to provide way more details about user activity without depending on database checks

![image](https://github.com/user-attachments/assets/8b4fcac6-f56f-4232-a332-c25d24f9a35c)

For the user profile, the visualization is simplified and helps to quickly track a recent change when sharing profiles (use case for many companies)

![image](https://github.com/user-attachments/assets/3a514127-f048-4832-a2e1-14e2056a63d8)

Inspired by this GItHub feature:
https://github.com/settings/security-log

I first created as tables, but the current version above might be better

old version as table:

![image](https://github.com/user-attachments/assets/abcb4ade-1fd3-4186-9c81-636ec89f7120)







---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Account > Account > My activity and see
3. Open Users > any user > See User activity
